.github/ISSUE_TEMPLATE/71_org_editor-recruit_reviewers.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "[Organizer -> Editor] Recruit Reviewers"
+about: Checklist for Associate Editor to fill out prior to review
+title: "[PRE-REVIEW] Recruit Reviewers"
+labels: pre-review
+assignees: ''
+---
+
+<!--
+## DO NOT EDIT THIS FILE OUTSIDE OF THE journalovi/jovi-workflows REPOSITORY
+##
+## This file is automatically updated in all repositories within the journalovi
+## Github organization whenever the version in journalovi/jovi-workflows is
+## changed, so any other edits will be overwritten. To update this file, make
+## a commit or pull request at https://github.com/journalovi/jovi-workflows
+-->
+
+### Instructions for Editor
+- Send out review requests using the [reviewer invitation email template](https://docs.google.com/document/d/15ttXvQg7g5JEk2Yzfd9eXSN6oIpc_A61qbuwygthuG0/edit?usp=sharing).
+  - Replace "LINK TO REVIEW ISSUE TEMPLATE" in the email with [this link](new?assignees=&labels=review&projects=&template=00_reviewer-review.yml&title=%5BREVIEW%5D+%3Ctitle%3E) (Right click to copy it.)
+- When a reviewer agrees to review, complete the corresponding steps for that reviewer below.
+- Close this issue when all reviewers have been recruited.
+
+### Recruit reviewers:
+- [ ] Reviewer 1 recruited.
+  - [ ] [Review Request Issue](new?assignees=&labels=pre-review&projects=&template=20_editor_reviewer-write_review.md&title=%5BPRE-REVIEW%5D+Review+Request) created and assigned to them.
```

Comment:
Change to action wording (instead of passive voice)

```
  - [ ] Create a [Review Request Issue](new?assignees=&labels=pre-review&projects=&template=20_editor_reviewer-write_review.md&title=%5BPRE-REVIEW%5D+Review+Request) and assign the reviewer to it (see how to assign them below)
```


There should be instructions on:
1. How to add reviewers to this GitHub repo. (From my test, people's Github IDs do not automatically pop up on the suggestion unless they are already a collaborator of the repo. However, I'm not 100% sure about this point.)
2. How to assign them once their Github ID is added.

If the AE cannot just add reviewer to the repo, may be the step of creating the Review Request Issue and assignment could be done by the person who added the reviewer.

Reply:
Yeah... to make it easier to both know who is assigned to what roles in a repo and to make it easier to mention people, I have:

- adjusted the tech chair checklist to have them add entries for OC, AE, and reviewers to the top of README.md
- adjusted the OC template for AE assignment to have the OC add the AE's github ID to the top of README.md and to give the AE admin access on the repo (so AEs appear in `@` autocomplete lists and so they can add reviewers in the next step)
- adjusted the AE template for reviewer assignment to have the AE add each reviewer's ID to the top of README.md and to give each reviewer explicit read access to the repo (rather than just relying on the repo being public). Giving them read access will make them appear in the autocomplete when you type `@` in an issue.

Reply:
I also changed the instructions to active voice, and re-ordered so that reviewers are added prior to being assigned to the issue (and linked to the relevant github documentation on assignment).